reference/docs-conceptual/whats-new/differences-from-windows-powershell.md

```diff
@@ -712,8 +987,13 @@ valid single value JSON `null` literal as `$null`.
 ### Web Cmdlets warn when `-Credential` is sent over unencrypted connections
 
 When using HTTP, content including passwords are sent as clear-text. This change is to not allow
-this by default and return an error if credentials are being passed in an insecure manner. Users
-can bypass this by using the `-AllowUnencryptedAuthentication` switch.
+this by default and return an error if credentials are being passed insecurely. Users can bypass
+this by using the `-AllowUnencryptedAuthentication` switch.
+
+### Make `-OutFile` parameter in web cmdlets to work like `-LiteralPath`
```

Comment:
Should this H3 have backticked text? I see it for several other headers in this section as well (not changed by this PR).

Reply:
Yeah, this is allowed.